### PR TITLE
Added CACHE_REVISION warnings

### PR DESCRIPTION
**Pull request by Stevoisiak**
I also slightly modified the CACHE_REVISION warnings for consistency.

### CODE REVIEW

**Comment by Buddybenj** [comment]
LGTM

**Comment by Stevoisiak** [comment]
@waddlesplash: Check my changed files. I did add the warning to Qt's GameFile.cpp

**Comment by Stevoisiak** [comment]
@waddlesplash Should I make a note that there are two separate caches? Maybe refer specifically to DolphinQt's CACHE_REVISION when relevant?

**Comment by Stevoisiak** [comment]
On second thought, it's probably better just to tell them apart by listing the source file.

File: Source/Core/DiscIO/VolumeCommon.cpp [hunk 1]
```diff
@@ -9,6 +9,7 @@
 #include "Common/Logging/Log.h"
 #include "DiscIO/Volume.h"
 
+// Increment CACHE_REVISION if the code below is modified (ISOFile.cpp)
```
Comment (waddlesplash) [inline Source/Core/DiscIO/VolumeCommon.cpp]: Need to mention GameFile.cpp here too

**Comment by waddlesplash** [comment]
LGTM

**Pull request merged**